goutil/goutil_test.go

```diff
@@ -72,6 +72,7 @@ func TestExpand(t *testing.T) {
 				"github.com/teamwork/utils/httputilx/header",
 				"github.com/teamwork/utils/imageutil",
 				"github.com/teamwork/utils/ioutilx",
+				"github.com/teamwork/utils/iputil",
```

Comment:
The naming schema has thus far been "[stdlibname]util", so if you want to stick with that if should be in netutil. Otherwise should probably also update the README if you want to let that go (I don't really care either way, just FYI).

Reply:
@ripexz Was the person who suggested this name.

Reply:
I told you to put in `netutils` first but you wanted a separate package 🤔